app/db_con.py

```diff
@@ -0,0 +1,11 @@
+import psycopg2
+
+DB_HOST =
+DB_USERNAME = 
+DB_PASS = 
+DB_NAME =
+DB_PORT =
+
+url = 
+
+con = psycopg2.connect(url)
```

Comment:
Add an end of line here, also where do the values for connection come from?

Reply:
I only set up PostgreSQL and psycopg2. I'll fill in those fields when I make the connection between the two and start using my endpoints through psycopg2 to edit the database

Reply:
Okay noted.